test/python/install_python311_setuptools_vulnerability.sh

```diff
@@ -79,8 +79,23 @@ checkVulnerableDir()
     fi
 }
 
+checkVulnerableFile()
+{
+    FILE=$1
+    VERSION=$2
+    
+    if [[ -f $FILE ]] ; then
+        echoStderr "❌ check for vulnerable setuptools version failed for python ${VERSION}."
+        return 1
+    else
+        echo "✅ Passed! Either the container does not have vulnerable version or vulnerable version specific file got removed."
+        return 0
+    fi
+}
+
 # only for 3.11
 checkVulnerableDir "/usr/local/py-utils/shared/lib/python3.11/site-packages/setuptools-65.5.0.dist-info" "3.11"
+checkVulnerableFile "/usr/local/lib/python3.11/ensurepip/_bundled/setuptools-65.5.0-py3-none-any.whl" "3.11"
```

Comment:
Can we add few more tests to validate that `pip` works fine?

Reply:
added tests to check pip works fine